src/app/openmrs-api/openmrs-api.module.ts

```diff
@@ -6,7 +6,7 @@ import { AppSettingsModule } from '../app-settings/app-settings.module';
 import { FormsResourceService } from './forms-resource.service';
 import { EncounterResourceService } from './encounter-resource.service';
 import { PatientResourceService } from './patient-resource.service';
-import { PatientSearchService } from '../patient-dashboard/patient-search/patient-search.service';
+import { PatientSearchService } from '../patient-search/patient-search.service';
```

Comment:
I believe this(patient search service) was added here by mistake in the first place during initial development. Should be moved to the patient search module